src/ipmanager/ui/templates/ui/single_group_view.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'ui/base.html' %}
+{% block content %}
+<h1> {{group.key}} </h1>
+
+<h4 style="margin-bottom: 0.5rem;">Group Information</h4>
```

Comment:
This (and the other `<h4>` tags) should be `<h2>` instead. For accessibility reasons, do not skip levels in the `h1`-`h6` tags:

> A common navigation technique for users of screen reading software is to quickly jump from heading to heading in order to determine the content of the page. Because of this, it is important to not skip one or more heading levels. Doing so may create confusion, as the person navigating this way may be left wondering where the missing heading is.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility